udata_hydra_csvapi/error.py

```diff
@@ -0,0 +1,14 @@
+import json
+from aiohttp import web
+
+
+class QueryException(web.HTTPException):
+    """Re-raise an exception from postgrest as aiohttp exception"""
+    def __init__(self, status, title, detail) -> None:
+        error_body = {
```

Comment:
It does not exactly follow our colleagues usage, having an encapsulating `errors` key:
```json
{
  "errors": [
    {
      "code": "code unique",
      "title": "message court à afficher dans l'interface de l'utilisateur",
      "detail": "message long qui explicite en détail l'erreur",
      "source": "qui a généré l'erreur",
      "meta": {
        # espace pour mettre d'autres infos au besoin
      }
    }
  ]
}
```